### PR TITLE
[CoreBundle] CLI imports improved

### PR DESCRIPTION
**Pull request by ngodfraind**

### CODE REVIEW

**Comment by LaurentGruber** [comment]
LGTM

File: main/core/Command/Import/CreateTextWidgetFromCsvCommand.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        $file = $input->getArgument('csv_widget_path');
+
+        $errors = $validator->validateValue($file, new CsvHomeTextWidget());
+
+        if (count($errors)) {
+            foreach ($errors as $error) {
+                $output->writeln("<error> {$error->getMessage()} </error>");
+            }
+            throw new \Exception('The csv file is incorrect');
+        }
```
Comment (stefk) [inline main/core/Command/Import/CreateTextWidgetFromCsvCommand.php]: Can't those csv commands be refactored? It seems most of them share around 50-60 lines of code (argument checking, interaction, error reporting, ...). That's a lot of copy/paste IMO.

**Comment by LaurentGruber** [comment]
LGTM

**Comment by ngodfraind** [comment]
The merge was pretty hard. I hope I didn't break anything.

**Comment by LaurentGruber** [comment]
LGTM

**Comment by ngodfraind** [comment]
LGTM

**Pull request merged**